lib/services/presets.go

```diff
@@ -166,6 +166,7 @@ func NewPresetEditorRole() types.Role {
 					types.NewRule(types.KindDiscoveryConfig, RW()),
 					types.NewRule(types.KindSecurityReport, append(RW(), types.VerbUse)),
 					types.NewRule(types.KindAuditQuery, append(RW(), types.VerbUse)),
+					types.NewRule(types.KindAccessGraph, append(RW(), types.VerbUse)),
```

Comment:
Is `Use` going to have meaning for access graph? Or are we just adding `VerbUse` to any new resource for completeness?

Reply:
No, that's a bug. I'll fix it before merging.